test/spec/clojure_smoke_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
+
+# Test both the released version of the Clojure buildpack as well as the code on main
+clojure_buildpacks = %w[
+  https://buildpack-registry.s3.amazonaws.com/buildpacks/heroku/clojure.tgz
```

Comment:
The buildpack aliases will work here too btw, eg:
https://github.com/heroku/heroku-buildpack-python/blob/b3e0b29a867ed6a9f8cb53341f713791a4fbab0d/spec/hatchet/pip_spec.rb#L7

Reply:
Oh, that's nice - less implementation details leaked that way. Thanks! I changed this in [5e2e807](https://github.com/heroku/heroku-buildpack-jvm-common/pull/328/commits/5e2e807398e07d9afcc5e7b03ca295499cc35cb2).